src/test/java/org/cactoos/list/JoinedTest.java

```diff
@@ -96,24 +99,31 @@ public void iterator() throws Exception {
     }
 
     @Test
-    public void toArray() throws Exception {
-        MatcherAssert.assertThat(
-            new Joined<Integer>(
-                new ListOf<>(11, 12),
-                new ListOf<>(13, 14)
-            ).toArray(),
-            new IsEqual<>(new ListOf<>(11, 12, 13, 14).toArray())
-        );
-    }
-
-    @Test(expected = UnsupportedOperationException.class)
     public void add() throws Exception {
-        new Joined<String>(new ListOf<String>("add0")).add("new add0");
+        final List<String> joined = new Joined<String>(
+            new ListOf<>("One"),
+            new ListOf<>("Two")
+        );
+        joined.add("Three");
+        new Assertion<>(
```

Comment:
@fanifieiev It is better to assure if the new list was properly joined by looking into its elements than just counting them in this case